### ISSUE TITLE
Missing method popup_message_toast (interface Z2UI5_IF_CLIENT)

### ISSUE THREAD

**Issue by chaouki-akir** [abap2UI5/abap2UI5#753]
Related to developer challenge : https://groups.community.sap.com/t5/application-development-discussions/sap-developer-challenge-open-source-abap-week-2/m-p/260727#M1372

The class created for the challenge few months ago doesn't compile any more today : 
Method "POPUP_MESSAGE_TOAST" is unknown or PROTECTED or PRIVATE.

This method seems to be missing interface Z2UI5_IF_CLIENT today.

Example of repo that are no more working today : 
https://github.com/chaouki-akir/developer_challenge2023/blob/main/week2/zcl_app_week_02
https://github.com/AndreaBorgia-Abo/abap2ui5-tests/blob/main/src/zcl_abo_abap2ui5_simpleapp.clas.abap

**Comment by choper725** [comment]
seems like an old version of abap2ui5, alot of refactoring was done since

**Comment by oblomov-dev** [comment]
yes if you install the actual abap2UI5 version the examples & solutions from the developer challenge are not working anymore.

but i recommend to install the newest abap2UI5 version and also install the samples repository:
https://github.com/abap2UI5/abap2UI5-samples
There you will find demos with all needed functions.

you can display the message toast for example now with the method:
client->message_toast_display

i also recommend to read the blog series to get a basic understanding how to create apps:
https://blogs.sap.com/2023/02/22/abap2ui5-development-of-ui5-apps-in-pure-abap-1-3/

Best regards.

**Comment by chaouki-akir** [comment]
Thank you. I will check that
